ACCESSIBILITY.md

```diff
@@ -0,0 +1,97 @@
+
+### 2.1 Web Accessibility
+
+_The Turing Way_ is committed to ensuring the web accessibility of our guides.
+
+As an open source community that relies on  projects and platforms for our underlying infrastructure, we acknowledge that we are limited in what standalone access-centered support we can offer. Our project is built with JupyterBook and using the Github platform. Both of these platforms have policies that address how they address the [W3C WAI's Web Content Accessibility Guidelines 2.0](https://www.w3.org/TR/WCAG/). You can read more about the the experience of a researcher at Accessible Notebooks Hackathon in ["Notebooks For All: Making Jupyter Notebooks More Accessible"](https://astrobites.org/2023/06/10/notebooks-for-all/) and about Github's work in the [Github Accessibility Report](https://accessibility.github.com/).
+
+As a community however, we are committed to ensuring broader accessibility of our resources through using this infrastructure through:
+- Choosing sufficient color contrast in our hand-drawn images that make our images accessible in the context of color-blindness
+- Using and employing appropriate and relevant alt text (also known as 'alternative text') for all of our images, charts, and data visualisations (an ongoing project with in the community)
+- Navigation without the need for a mouse or track pad during either the reading process on JupyterBook and/or contributing process on Github
+
+### 2.2 Community Platform Accessibility
+
+_The Turing Way_ is committed to ensuring our accessibility as a community on all the platforms we use.
+
+In our Community Handbook, we describe these practices more specifically and in more detail. But as a baseline, our community is dedicated to enabling access across the various platforms we use through:
+- Providing diverse opportunities and platforms for people to contribute to _The Turing Way_ despite barriers or difficulties associated with one platform or another
+- Setting and embodying standards for writing and contribution that understand the way in which language may exclude as well as include people in data science, research, and other collaborative projects
+
+### 2.3 Community Event Accessibility
+
+As a community, we frequently organise events for a variety of purposes: for coworking, trainings, workshops, panels, conference talks, and more.
+
+In our Community Handbook, we describe these practices more specifically and in more detail. As a baseline however, we are committed to enabling accessibility through:
+- Supplying and recording transcripts for audio and closed captioning of video content
+- Enabling wide access to our program context across a variety of platforms
+- Uploading slides and/or presentation materials on Zenodo or other platforms to enable referencing
```

Comment:
```suggestion
In our Community Handbook we describe these practices more specifically and in more detail.
As a baseline however, we are committed to enabling accessibility through:

- Supplying and recording transcripts for audio and closed captioning of video content
- Enabling wide access to our program context across a variety of platforms
- Uploading slides and/or presentation materials on Zenodo or other platforms to enable referencing
```

Reply:
Thank you for the formatting suggest @JimMadge!